doc/en/mooncake-store-preview.md

```diff
@@ -382,6 +383,13 @@ To avoid data conflicts, a per-object lease will be granted whenever an `ExistKe
 
 The default lease TTL is 200 ms and is configurable via a startup parameter of `master_service`.
 
+### Multi-layer Storage Support
+When a user specifies the environment variable `MOONCAKE_STORAGE_ROOT_DIR` at client startup, and the path is a valid existing directory, the client-side data persistence feature will be activated. During initialization, the client requests a `cluster_id` from the master. This ID can be specified when initializing the master; if not provided, the default value `mooncake_cluster` will be used. The root directory for persistence is then set to `<MOONCAKE_STORAGE_ROOT_DIR>/<cluster_id>`. Note that when using DFS, each client must specify the corresponding DFS mount directory to enable data sharing across SSDs.
```

Comment:
1. It would be better to write out the full name of DFS the first time it is mentioned.
2. It would be better to first introduce the concept and mechanism of Multi-layer Storage Support, and then describe the detailed aspects, such as how to enable it and how to configure directory names.